test/Altinn.App.Core.Tests/Features/Maskinporten/MaskinportenClientTest.cs

```diff
@@ -265,16 +274,49 @@ public async Task GetAccessToken_ThrowsExceptionWhenTokenIsExpired(string varian
         // Act
         Func<Task> act1 = async () =>
         {
-            await fixture.Client(variant).GetAccessToken(["scope1", "scope2"]);
+            await fixture.Client(variant).GetAccessToken(["scope"]);
         };
         Func<Task> act2 = async () =>
         {
-            await fixture.Client(variant).GetAltinnExchangedToken(["scope1", "scope2"]);
+            await fixture.Client(variant).GetAltinnExchangedToken(["scope"]);
         };
 
         // Assert
-        await act1.Should().ThrowAsync<MaskinportenTokenExpiredException>();
-        await act2.Should().ThrowAsync<MaskinportenTokenExpiredException>();
+        await Assert.ThrowsAsync<MaskinportenTokenExpiredException>(act1);
+        await Assert.ThrowsAsync<MaskinportenTokenExpiredException>(act2);
+    }
+
+    [Theory]
+    [MemberData(nameof(Variants))]
+    public async Task TokenCache_ReusesObjects_DoesNotSerialize(string variant)
```

Comment:
Not sure I quite got the name here. Could this be an alternative? 🤷‍♀️ 
```suggestion
    public async Task TokenCache_Returns_SameInstanceForIdenticalRequests(string variant)
```

Reply:
I guess what I meant was that the HybridCache wants really badly to serialize objects put into cache, and deserialize them on the way out again. That's why the `TokenCacheEntry` is annotated with `ImmutableObject(true)`, to prevent this.

But that said, I'm not arguing that the intial test name was in any way _good_ 😆